modules/op-about-workspace.adoc

```diff
@@ -43,27 +43,27 @@ spec:
 ...
   tasks: <2>
   - name: build-image
-   taskRef:
-     name: buildah
-     kind: ClusterTask
-   params:
-   - name: TLSVERIFY
-     value: "false"
-   - name: IMAGE
-     value: $(params.IMAGE)
-   workspaces: <3>
-   - name: source <4>
-     workspace: shared-workspace <5>
-   runAfter:
-   - fetch-repository
- - name: apply-manifests
-   taskRef:
-     name: apply-manifests
-   workspaces: <6>
-   - name: source
-     workspace: shared-workspace
-   runAfter:
-    - build-image
+    taskRef:
```

Comment:
@Preeticp, do you know who from QE can confirm this YAML update? Thanks!

Reply:
@VeereshAradhya can you please take a look and add an LGTM if this looks good?

Reply:
LGTM